tools/buildsys/src/manifest.rs

```diff
@@ -352,6 +381,9 @@ pub(crate) struct ExternalFile {
     pub(crate) path: Option<PathBuf>,
     pub(crate) sha512: String,
     pub(crate) url: String,
+    pub(crate) bundle_modules: Option<Vec<String>>,
```

Comment:
We can enforce stronger typing here using an enum, like `SupportedArch` and `GrubFeature` do.

(Then use that enum with a match statement in `main.rs` to ensure that all known values are handled.)

Reply:
👍🏼  Added enum for this struct field